mne_bids/tests/test_mne_bids.py

```diff
@@ -112,9 +116,18 @@ def test_ctf():
 
     raw_to_bids(subject_id=subject_id, session_id=session_id, run=run,
                 task=task, raw_file=raw_fname, output_path=output_path,
-                overwrite=True)
+                overwrite='overwrite')
     cmd = ['bids-validator', output_path]
     run_subprocess(cmd, shell=shell)
+
+    # test to check that the 'error' overwrite parameter works correctly
+    with pytest.raises(OSError):
+        # FIXME: is there a way in py2 to get this to detect the correct
+        # sub-type of OSError?
```

Comment:
is this still a problem?

Reply:
`FileExistsError` doesn't exist for python 2, so I had to change all the errors raised to an `OSError`
The issue here isn't too big, although if a *different* OSError is raised somehow pytest will not be able to differentiate between the raised error and the one we expect (which has error number 17)

Reply:
why can't you just replace `OSError` by `errno.EExist` ?

Reply:
`errno.EEXIST` is just an int (=17)

Reply:
I see. Maybe you want to then match the string of your error using the "match" argument?

Reply:
Yeah, we could do this if we then specify the same string for each error, however this wouldn't pick up the error raised by this:
https://github.com/mne-tools/mne-bids/blob/d501b9daf20e50e22d26504293f9cc78dd232c4c/mne_bids/mne_bids.py#L726

Reply:
you can't give a substring? I think [that works](https://github.com/mne-tools/mne-python/blob/master/mne/tests/test_epochs.py#L539)